### ISSUE TITLE
One type of ColorKeyFrame class missing

### ISSUE THREAD

**Issue by kimmoli** [dotnet/docs#5990]
In the Remarks chapter, paragraph starting "There are three types of ColorKeyFrame classes, ..." is missing the fourth type, EasingColorKeyFrame.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3c7092e2-e011-7332-03b0-924007a9cebf
* Version Independent ID: 7095a2d4-ad64-ae41-7195-0de7ade7d59d
* Content: [ColorAnimationUsingKeyFrames Class (System.Windows.Media.Animation)](https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.animation.coloranimationusingkeyframes?view=netframework-4.7.2)
* Content Source: [xml/System.Windows.Media.Animation/ColorAnimationUsingKeyFrames.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Media.Animation/ColorAnimationUsingKeyFrames.xml)
* Product: **.net**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by mairaw** [comment]
You're right @kimmoli. Thanks for the feedback. We'll add the missing one.
